localstorage.js

```diff
@@ -0,0 +1,9 @@
+const form = document.querySelector('form');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const formData = new FormData(form);
+
+  localStorage.setItem('formData', JSON.stringify(Object.fromEntries(formData.entries())));
+});
```

Comment:
**1-** Currently, there is an error in the logic as the form Identifier or variable is already declared already in the `form.js file` causing the form not to submit or causing the error. 

![image](https://user-images.githubusercontent.com/94223367/233445814-f2b5759c-6405-4c4c-aea6-f172cf926bdd.png)



**2-** Moreover, when I click submit button it only then updates the local state else what you need to do is the following:-
> Whenever the user adds something in form inputs it should be updated at once in the local storage else when the user clicks submit button.

> When the user reloads the page even without submitting or after submitting the data he entered should be collected from local storage and shown in input tags.

> This is how the local sotragwe object needs to be:-


```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

this is what is is tight now:-

```
{
    "area": "Hy its me ABDULALI"
}
```

Consider resolving the issues to meet the project requirements.